includes/class-wc-stripe-checkout-session-lifecycle.php

```
@@ -0,0 +1,244 @@
+
+		if ( ! self::is_adaptive_pricing_enabled() ) {
+			return $manager->mark_failed( WC_Stripe_Checkout_Session_Context::get_unavailable_message() );
+		}
+
+		try {
+			return $manager->synchronize();
+		} catch ( Exception $e ) {
+			WC_Stripe_Logger::error( 'Native Store API checkout session synchronization failed.', [ 'error_message' => $e->getMessage() ] );
+			return $manager->mark_failed( $e->getMessage() );
+		}
+	}
```

PHPMD (2.15.0)

[error] The method get_store_api_data is not named in camelCase. (undefined)

(CamelCaseMethodName)